src/core/term.ml

```diff
@@ -232,11 +233,12 @@ let rec pp_term : term pp = let out = Format.fprintf in fun ppf t ->
   | Meta(m,ts) -> out ppf "?%a[%a]" pp_meta m pp_terms ts
   | Patt _ -> out ppf "<Patt>"
   | TEnv _ -> out ppf "<TEnv>"
-  | Wild -> out ppf "_"
+  | Wild   -> out ppf "_"
   | TRef _ -> out ppf "<TRef>"
   | LLet(a,t,u) ->
       let x, u = Bindlib.unbind u in
       out ppf "let %a : %a ≔ %a in %a" pp_var x pp_term a pp_term t pp_term u
+  | Plac _ -> out ppf "_"
```

Comment:
Should be improved following #697 (see case of Patt). Term.pp_term is for debug and should print every thing useful for debug.